march_state_machine/src/march_state_machine/gaits/tilted_path_left_knee_bend_sm.py

```diff
@@ -0,0 +1,30 @@
+import smach
+
+from march_state_machine.state_machines.step_state_machine import StepStateMachine
+from march_state_machine.states.idle_state import IdleState
+
+
+def create():
+    sm_tilted_path_left_knee_bend = smach.StateMachine(outcomes=['succeeded', 'preempted', 'failed'])
```

Comment:
I think this is the problem, forgot to mention that :sweat_smile: 
```suggestion
    sm_tilted_path_left_knee_bend = smach.StateMachine(outcomes=['succeeded', 'preempted', 'failed', 'rejected'])
```